2017-12-16T06:34:55.927Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))
    at /home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/fabric-client/node_modules/grpc/src/node/src/client.js:434:17 {}$
2017-12-16T06:34:55.930Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at channel.queryByChaincode.then.catch (/home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:749:34)"}$
2017-12-16T06:34:55.930Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at _checkRuntimeVersions.then.catch (/home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:663:34)"}$
2017-12-16T07:55:02.177Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)
    at /home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/fabric-client/node_modules/grpc/src/node/src/client.js:434:17 {}$
2017-12-16T07:55:02.181Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)","stack":"Error: Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)\n    at _initializeChannel.then.then.then.then.catch (/home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:837:34)"}$
2017-12-16T07:58:53.744Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)
    at /home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/fabric-client/node_modules/grpc/src/node/src/client.js:434:17 {}$
2017-12-16T07:58:53.745Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)","stack":"Error: Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)\n    at _initializeChannel.then.then.then.then.catch (/home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:837:34)"}$
2017-12-16T07:58:58.386Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)
    at /home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/fabric-client/node_modules/grpc/src/node/src/client.js:434:17 {}$
2017-12-16T07:58:58.387Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)","stack":"Error: Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)\n    at _initializeChannel.then.then.then.then.catch (/home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:837:34)"}$
2017-12-16T07:59:17.268Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)
    at /home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/fabric-client/node_modules/grpc/src/node/src/client.js:434:17 {}$
2017-12-16T07:59:17.268Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)","stack":"Error: Error trying invoke business network. Error: chaincode error (status: 500, message: Error: Failed to add object with ID '1' as the object already exists)\n    at _initializeChannel.then.then.then.then.catch (/home/ubuntu/.nvm/versions/node/v6.11.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:837:34)"}$
